src/routes/about/+page.svelte

```diff
@@ -38,8 +38,13 @@
 <h2>Constitution</h2>
 <p>
 	As required by the University of Michigan and the SOAR (Student Organization and Recognition)
-	Initiative, our club constitution can be found on <a
-		href="https://maizepages.umich.edu/organization/kendoclub"
-		>Maize pages.
-	</a>
+	Initiative, our club constitution can be found on
+	<a href="https://maizepages.umich.edu/organization/kendoclub">Maize pages</a>.
 </p>
+
+<style>
+	.motto {
+		font-size: 1.5em;
```

Comment:
Did you test other sizes? It seams like it could be a little larger if you wanted.

Reply:
I didn't test many other sizes, I'll try some more.
I was worried about it being the same or similar size as other headers and making the structure of the page less clear.

Reply:
I think 2.5em looks pretty good, what do you think?
![image](https://github.com/kendoclub-at-umich/website/assets/75005450/318ed0c3-654b-406d-ab15-5210ec1c57d7)

Reply:
Yup makes sense. Added in [7acc634](https://github.com/kendoclub-at-umich/website/pull/45/commits/7acc634635321731321ccfdd7769102b9b03f7fc)